frontend/views/containers/proposals/Mincome.vue

```diff
@@ -126,16 +126,17 @@ export default {
       }
 
       try {
+        this.foo.dumbError = 'force-an-error' // DELETE THIS BEFORE MERGE
         const updatedSettings = await sbp(
           'gi.contracts/group/updateSettings/create',
           { mincomeAmount },
           this.currentGroupId
         )
         await sbp('backend/publishLogEntry', updatedSettings)
         this.$refs.proposal.close()
-      } catch (e) {
-        console.error(`Failed to change mincome to ${mincomeAmount}`, e.message)
-        this.$refs.formMsg.danger(L('Failed to change mincome {codeError}', { codeError: e.message }))
+      } catch (error) {
+        console.error('Mincome.vue submit() error:', error)
+        this.$refs.formMsg.danger(L('Failed to change mincome. {reportError}', LError(error)))
```

Comment:
Should read:

```js
this.$refs.formMsg.danger(L('Error changing mincome: {error}. {reportError}', LError(error)))
```

Reply:
@taoeffect, here's a code review tip:

If you see that the same error is made multiple times, prefer to just leave one comment about it and remember the author to review it in the other places, too. Adding redundant comments only creates noise and might be contra-productive.

Reply:
OK, I will do my best to remember that for you. Different people seem to prefer (or need) different styles of code review.